pkg/controller/importconfig/cluster_info.go

```diff
@@ -145,10 +145,6 @@ func getKubeAPIServerCertificate(ctx context.Context, kubeClient kubernetes.Inte
 		return nil, err
 	}
 
-	if secret.Type != corev1.SecretTypeTLS {
```

Comment:
should we check all types we supported or just remove the check？

Reply:
just remove the type check, and we check tls key below